examples/src/index.js

```diff
@@ -143,8 +142,7 @@ class ReactDemo extends React.Component {
               ref={this.captcha}
               onVerify={this.onVerifyCaptcha}
               languageOverride={this.languageOverride}
-              sitekey="917ba1eb-0b37-486e-9c90-39f3cb7b2579"
```

Comment:
There are 4 more other places where we still use this sitekey in the `examples/src/index.js`.